src/components/edit-user.html

```diff
@@ -37,31 +34,93 @@
       }
     </style>
     <h3>Edit User Component</h3>
-    <form>
-      <fieldset>
-        <div>
-          <label for="firstName">First name:</label>
-          <input type="text" name="firstName" id="firstName">
-        </div>
-        <div>
-          <label for="lastName">Last name:</label>
-          <input type="text" name="lastName" id="lastName">
-        </div>
-        <div>
-          <label for="phone">Phone:</label>
-          <input type="text" name="phone" id="phone">
-        </div>
-        <div>
-          <label for="email">Email:</label>
-          <input type="text" name="email" id="email">
-        </div>
-        <button type="submit">Save Edit</button>
-      </fieldset>
-    </form>
+    <template is="dom-if" if="[[editSelected]]">
+      <table>
+        <tr>
+          <td>ID :</td>
+          <td>[[user.id]]</td>
+        </tr>
+        <tr>
+          <td><label for="name">Name :</label></td>
+          <td>[[user.firstName]] [[user.lastName]]</td>
+        </tr>
+        <tr>
+          <td><label for="phone">Phone :</label></td>
+          <td>[[user.phoneNumber]]</td>
+        </tr>
+        <tr>
+          <td><label for="email">Email :</label></td>
+          <td>[[user.email]]</td>
+        </tr>
+        <tr>
+          <td><button type="button" on-click="editUserProfile">[[editButtonText(editSelected)]]</button></td>
+        </tr>
+      </table>
+    </template>
+    <template is="dom-if" if="[[!editSelected]]">
+      <table>
+        <tr>
+          <td><label for="firstName">First name :</label></td>
+          <td><input type="text" name="firstName" id="firstName" value="{{user.firstName}}" on-input="updateFirstName"></td>
+        </tr>
+        <tr>
+          <td><label for="lastName">Last name :</label></td>
+          <td><input type="text" name="lastName" id="lastName" value="{{user.lastName}}" on-input="updateLastName"></td>
+        </tr>
+        <tr>
+          <td><label for="phone">Phone :</label></td>
+          <td><input type="text" name="phone" id="phone" value="{{user.phoneNumber}}" on-input="updatePhoneNumber"></td>
+        </tr>
+        <tr>
+          <td><label for="email">Email :</label></td>
+          <td><input type="text" name="email" id="email" value="{{user.email}}" on-input="updateEmail"></td>
+        </tr>
+        <tr>
+          <td><button type="button" on-click="editUserProfile">[[editButtonText(editSelected)]]</button></td>
+        </tr>
+      </table>
+    </template>
   </template>
   <script type="module">
     class EditUser extends Polymer.Element {
       static get is() { return 'edit-user'; }
+
+      static get properties() {
+        return {
+          editSelected: {
+            type: Boolean,
+            value: true
+          },
+          user: {
+            type: Object,
+          }
+        };
+      }
+
+      editUserProfile(event) {
+        // event.preventDefault();
```

Comment:
Did you want this commented out?

Reply:
I'll get rid of that actually, it was to fix a problem with the save button causing a page refresh